pandas/tests/resample/test_period_index.py

```diff
@@ -489,16 +479,17 @@ def test_cant_fill_missing_dups(self):
             s.resample("Y").ffill()
 
     @pytest.mark.parametrize("freq", ["5min"])
```

Comment:
Could you inline this parameter since it's the only one?

Reply:
done